### ISSUE TITLE
conda package

### ISSUE THREAD

**Issue by markusweimer** [microsoft/onnxruntime#7056]
Are there plans to create a conda package for onnxruntime, specifically on `conda-forge`?

**Background:** We are working on the next version of the default Python environment on Azure Synapse. We'd very much like to have `onnxruntime` in that default environment. We have chosen `conda` as our environment manager. For simplicity and stability reasons, we prefer as few `pip` dependencies in the default installation as possible.

**Comment by askhade** [comment]
Right now we dont have any plans to release conda packages for ort inferencing.
 @faxu FYI

**Comment by xhochy** [comment]
Opened a PR for this at https://github.com/conda-forge/staged-recipes/pull/14552

**Comment by xhochy** [comment]
The package is now available on conda-forge, thus this issue can be closed.

**Comment by markusweimer** [comment]
Thanks!

**Comment by zooba** [comment]
Correct me if I'm wrong, but the conda-forge recipe has no Cuda references, so does that mean it's only the CPU version? No GPU acceleration?

**Comment by xhochy** [comment]
Yes, currently the package is only the CPU version. I opened a pull request that adds a CUDA flavor: https://github.com/conda-forge/onnxruntime-feedstock/pull/7

**Comment by irizwanihaque** [comment]
The onnxruntime on conda-forge is not available on windows (win-32/win-64) channel. Can you please get this added as well.

Trying to install from conda-forge gives following error:

PackagesNotFoundError: The following packages are not available from current channels:

  - onnxruntime

Current channels:

  - https://conda.anaconda.org/conda-forge/win-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

**Comment by roachsinai** [comment]
> The onnxruntime on conda-forge is not available on windows (win-32/win-64) channel. Can you please get this added as well.
> 
> Trying to install from conda-forge gives following error:
> 
> PackagesNotFoundError: The following packages are not available from current channels:
> 
> * onnxruntime
> 
> Current channels:
> 
> * [conda.anaconda.org/conda-forge/win-64](https://conda.anaconda.org/conda-forge/win-64)
> * [conda.anaconda.org/conda-forge/noarch](https://conda.anaconda.org/conda-forge/noarch)
> * [repo.anaconda.com/pkgs/main/win-64](https://repo.anaconda.com/pkgs/main/win-64)
> * [repo.anaconda.com/pkgs/main/noarch](https://repo.anaconda.com/pkgs/main/noarch)
> * [repo.anaconda.com/pkgs/r/win-64](https://repo.anaconda.com/pkgs/r/win-64)
> * [repo.anaconda.com/pkgs/r/noarch](https://repo.anaconda.com/pkgs/r/noarch)
> * [repo.anaconda.com/pkgs/msys2/win-64](https://repo.anaconda.com/pkgs/msys2/win-64)
> * [repo.anaconda.com/pkgs/msys2/noarch](https://repo.anaconda.com/pkgs/msys2/noarch)

same issue.

**Comment by garymm** [comment]
Please don't ask here about the conda-forge package. It has its own repo where you can report issues: https://github.com/conda-forge/onnxruntime-feedstock/

**Comment by roachsinai** [comment]
> Please don't ask here about the conda-forge package. It has its own repo where you can report issues: [conda-forge/onnxruntime-feedstock](https://github.com/conda-forge/onnxruntime-feedstock/)

Thanks for your reply, sorry for bother.
